jupyter_console/app.py

```diff
@@ -119,7 +120,7 @@ def handle_sigint(self, *args):
             if self.kernel_manager:
                 self.kernel_manager.interrupt_kernel()
             else:
-                self.shell.write_err('\n')
+                print("\n", file=sys.stderr, end="\n")
```

Comment:
This will now print two newlines, whereas it was only one before. Can one of them be removed?